ansible/roles/data-pipeline/tasks/main.yml

```diff
@@ -0,0 +1,172 @@
+---
+# tasks file for data-pipeline
+- name: Git checkout
+  git:
+    repo: 'https://github.com/Sunbird-Ed/ml-analytics-service.git'
+    dest: "/opt/sparkjobs/ml-analytics-service"
+    version: main
+
+- name: CREATE THE USER
+  user:
+    name: "{{ USER }}"
+    state: present
+    home: "/home/{{ USER }}"
+    shell: /bin/bash
+
+- name: Ensure the locale exists
+  locale_gen:
+    name: en_US.UTF-8
+    state: present
+
+- name: set as default locale
```

Comment:
can't we combine this with above one?

Reply:
both are different modules

Reply:
locale_gen module will accept only two parameters  (name and state) 
so we may not be able to merge those two tasks